hw/dv/tools/dvsim/common_sim_cfg.hjson

```diff
@@ -142,6 +142,7 @@
   // Project defaults for Xcelium
   xcelium_cov_cfg_file: "{dv_root}/tools/xcelium/xcelium.ccf"
   xcelium_cov_refine_files: []
+  xcelium_unr_cfg_file:  "{dv_root}/tools/xcelium/unr.cfg"
```

Comment:
this script defines the clock and reset used. as well as the report verbosity
should be picked up by the unr_run_opts (already added)

Reply:
sorry I did it is there now